src/components/layout/KoroSection/index.tsx

```diff
@@ -1,28 +1,65 @@
 import classnames from 'classnames';
```

Comment:
Why is this file called as `index.tsx`? Would it be a huge thing to change as `KoroSection.tsx`?